development/generate_credits.py

```diff
@@ -26,7 +26,9 @@
 
 sections = ['UH-Team_New', 'UH-Team_Old', 'Patchers', 'Translators', 'Packagers', 'Special Thanks']
 section_widgets = {s: 'credits_' + s.lower() for s in sections}
-section_widgets.update({'UH-Team-2016/2017': 'credits_team_2016','UH-Team-2015': 'credits_team_2015', 'Special Thanks': 'credits_thanks'})
+section_widgets.update({'UH-Team-2016/2017': 'credits_team_2016',
+						'UH-Team-2015': 'credits_team_2015',
+						'Special Thanks': 'credits_thanks'})
```

Comment:
As @squiddy observed, it's more of a GitHub thing. However, the solution you proposed looks good.